definitions/json/CaseEvent/CaseEvent-WA.json

```diff
@@ -75,8 +75,34 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "ShowSummary": "N",
+    "SignificantEvent": "Yes",
+    "TTLIncrement": "365",
```

Comment:
Similar to the comment for ET1 Repped, might be better to break this off into a prod and non prod variant